wiring/src/spark_wiring_wifitester.cpp

```diff
@@ -392,7 +392,7 @@ void WiFiTester::checkWifiSerial(char c) {
             if (ok) {
                 if (!strcmp("ALL", parts[1])) {
 #if PLATFORM_ID == PLATFORM_XENON || PLATFORM_ID == PLATFORM_ARGON || PLATFORM_ID == PLATFORM_BORON || \
-    PLATFORM_ID == PLATFORM_XSOM || PLATFORM_ID == PLATFORM_ASOM || PLATFORM_ID == PLATFORM_BSOM
+    PLATFORM_ID == PLATFORM_BSOM
```

Comment:
Can we use `#if HAL_PLATFORM_NRF52840` instead? I think it's also applicable for B5SoM.

Reply:
I think we need to review this part honestly for each platform, because it's not clear whether A6-A7 and D8 should be supported for each particular platform.